kernel/src/test/scala/org/apache/toree/kernel/protocol/v5/stream/KernelInputStreamSpec.scala

```diff
@@ -65,6 +65,9 @@ class KernelInputStreamSpec
     // set of data
     doReturn(system.actorSelection(fakeInputOutputHandlerActor.path.toString))
       .when(mockActorLoader).load(MessageType.Incoming.InputReply)
+    // Allow time for the actors to start. This avoids read() hanging forever
+    // when running tests in gradle.
+    Thread.sleep(100)
```

Comment:
I do, too. The problem is that this test is flaky on my machine without it. We need to take a closer look at the flaky test, but I don't think this PR is the place to fix it. How about I open an issue for it and we solve the problem separately?